src/base/bittorrent/torrenthandle.cpp

```diff
@@ -915,26 +915,29 @@ int TorrentHandle::leechsCount() const
 
 int TorrentHandle::totalSeedsCount() const
 {
-    return m_nativeStatus.list_seeds;
+    return (m_nativeStatus.num_complete > 0) ? m_nativeStatus.num_complete : m_nativeStatus.list_seeds;
```

Comment:
Sorry if I missed something from the various comments, but: What if know peers that the tracker doesn't (eg from DHT, LSD)?
I realize that there isn't a correct way, to decide either one. So you opted to use the tracker scrape info as more reliable?